dangerfile.js

```diff
@@ -0,0 +1,93 @@
+const { danger, markdown } = require('danger');
+const fs = require('fs').promises;
+const gzipSize = require('gzip-size');
+const git = require('simple-git/promise')(__dirname);
+
+// https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript
+const formatBytes = (bytes, decimals = 2) => {
+	if (bytes === 0) return '0 Bytes';
+
+	const k = 1000;
+	const dm = decimals < 0 ? 0 : decimals;
+	const sizes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+	const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+}
+
+const maybePlus = (from, to) => from < to ? "+" : "";
+
+const absDiff = (from, to) => {
+	if (from === to) {
+		return formatBytes(0);
+	}
+
+	return `${maybePlus(from, to)}${formatBytes(to - from)}`;
+}
+
+const percDiff =  (from, to) => {
+	if (from === to) {
+		return '0%';
+	}
+
+	return `${maybePlus(from, to)}${
+		(100 * (to - from) / ((from + to) / 2 )).toFixed(2)
```

Comment:
I think that we should round the percentages to whole numbers. Sub-percent changes don't matter anyway.
```suggestion
		(100 * (to - from) / ((from + to) / 2 )).toFixed()
```

Reply:
I disagree with this a bit. I think we're likely to see changes be in the 1% range where I'd like to know the % difference with a bit more granularity, so I'd like to keep the 2 decimal places.

Reply:
Ok, but let's turn it down to only 1 decimal digit. Most of our files are 1 to 2KB gzipped, so 0.01% represents 0.1 to 0.2 bytes.